frontend/src/stores/map.js

```diff
@@ -131,6 +131,17 @@ export const useMapStore = defineStore('map', () => {
       keepInView: true
     })
     currentFilteredData.value.push(feature)
+
+    layer.on('click', () => {
+      if (typeof window !== 'undefined' && window.heap) {
+        console.log('Heap is available.', feature.properties.location.long_name)
```

Comment:
I would remove these log statements unless you think they are valuable to the user.